youtube_dl/extractor/aqstream.py

```diff
@@ -0,0 +1,112 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import base64
+import hashlib
+import random
+from .common import InfoExtractor
+from ..aes import aes_cbc_decrypt
+from ..utils import (
+    ExtractorError,
+    bytes_to_intlist,
+    intlist_to_bytes,
+    decode_pkcs7,
+    urlencode_postdata,
+    remove_start,
+)
+
+
+class AqstreamIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?aqstream\.com/[-\w]+/(?P<id>[-\w]+)'
```

Comment:
Looks like at least `+` and `&` are also possible: `http://aqstream.com/mbcst/MBC-Star&`.